articles/quickstart/spa/angular-next/01-login.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login to an Angular application using Auth0.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - angular
+  - login
+github:
+    path: 01-Login
+contentType: tutorial
+useCase: quickstart
+---
+
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+<%= include('../_includes/_getting_started', { library: 'Angular', callback: 'http://localhost:4200', returnTo: 'http://localhost:4200', webOriginUrl: 'http://localhost:4200', showLogoutInfo: true, showWebOriginInfo: true, new_js_sdk: true, show_install_info: false }) %>
+
+## Install the Auth0 Angular SDK
+
+Run the following command within your project directory to install the Auth0 Angular SDK:
+
+```bash
+npm install @auth0/auth0-angular
+```
+
+The SDK exposes several types that help you integrate Auth0 with your Angular application idiomatically, including a module and an authentication service.
+
+### Register and configure `AuthModule`
+
+The SDK exports a module with the components and services used to perform the authentication. Import this module into your application module so that they can be accessed through Angular's dependency injection framework:
+
+```javascript
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { AppComponent } from './app.component';
+
+// Import the module from the SDK
+import { AuthModule } from '@auth0/auth0-angular';
+
+@NgModule({
+  declarations: [AppComponent],
+  imports: [
+    BrowserModule,
+
+    // Import the module into the application, with configuration
+    AuthModule.forRoot({
+      domain: 'YOUR_AUTH0_DOMAIN',
+      clientId: 'YOUR_AUTH0_CLIENT_ID',
+      redirectUri: window.location.origin
```

Comment:
given this https://juristr.com/blog/2016/09/ng2-get-window-ref/ that you shared earlier, should we remove this line or at least inject the service privately instead of using the global?
Note that below for returnTo (logout) you are using localhost:4200 instead of this

Reply:
You can't inject anything at this point so I would be in favour of removing it, considering we also default it in the SDK.

Reply:
Done.